src/include/reconcile.h

```diff
@@ -384,3 +386,13 @@ typedef struct {
         (upd_select)->no_ts_tombstone = false;  \
         WT_TIME_WINDOW_INIT(&(upd_select)->tw); \
     } while (0)
+
+/* Called when building the leaf disk image. */
+#define WT_TRY_BUILD_DELTA_LEAF(session, r)                                                     \
+    F_ISSET(S2BT(session), WT_BTREE_PAGE_DELTA) && !F_ISSET(S2C(session), WT_CONN_IN_MEMORY) && \
+      (r)->multi_next == 0 && !r->ovfl_items && (r->ref->page)->dsk != NULL
+
+/* Called when writing the leaf disk image. */
+#define WT_BUILD_DELTA_LEAF(session, r)                                                         \
```

Comment:
The difference between `TRY` and non-`TRY` is not clear to me, why do we care about `multi->next`?

Reply:
the number of multi->next indicates whether the page is split or not. When we are still building the first disk image, multi->next is 0. After we build the first disk-image, it becomes 1.